core/src/mindustry/entities/Damage.java

```diff
@@ -334,14 +334,14 @@ public static void damage(Team team, float x, float y, float radius, float damag
     /** Applies a status effect to all enemy units in a range. */
     public static void status(Team team, float x, float y, float radius, StatusEffect effect, float duration, boolean air, boolean ground){
         Cons<Unit> cons = entity -> {
-            if(entity.team == team || !entity.within(x, y, radius) || (entity.isFlying() && !air) || (entity.isGrounded() && !ground)){
+            if(entity.team == team || !entity.within(x, y, radius + entity.type.hitSize/2) || (entity.isFlying() && !air) || (entity.isGrounded() && !ground)){
                 return;
             }
 
             entity.apply(effect, duration);
         };
 
-        rect.setSize(radius * 2).setCenter(x, y);
+        rect.setSize((radius + 100) * 2).setCenter(x, y);
```

Comment:
All queries take hitbox size into account when checking collisions; this shouldn't be necessary. Same goes for the other occurrence of this.

Reply:
fixing